### PR TITLE
New changes for the README

### PR DESCRIPTION
**Pull request by abhiwalia15**
Added tagline and introduction with infographic images for the new README

## Description
Started designing and working on the changes for the new README for [PySyft ](https://github.com/OpenMined/PySyft)repository. 

## Affected Dependencies
The current README can be found [here](https://github.com/OpenMined/PySyft/blob/dev/README.md).

Issue #6593 will also be included in this README.

## How has this been tested?
- README is grammatically correct.
- This is still a work in progress.

## Checklist
- [+] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [ ] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [+] I have labelled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [+] My changes are covered by tests

### CODE REVIEW

File: README_NEW.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+For example, PySyft can improve your product based on customers' private data without them needing to disclose their sensitive information. Researchers will make medical advances without patients disclosing medical information. It also means 1000x more data in every domain. PySyft achieves this using [Federated Learning](https://ai.googleblog.com/2017/04/federated-learning-collaborative.html) and [Encrypted Computation](https://en.wikipedia.org/wiki/Homomorphic_encryption) and is currently under development. 
+
+
+<h5 align="center">
+    <img src="packages/syft/docs/img/Working_infographic.gif" alt="working_infographic" width="400">
+</h5>
+ 
+
+
+Data Owners provide (digital) datasets which they would like to make available for study by an outside party they may or may not fully trust has good intentions.
```
Comment (abhiwalia15) [inline README_NEW.md:38]: remove digital

File: README_NEW.md [hunk 2]
```diff
@@ -30,28 +30,23 @@ For example, PySyft can improve your product based on customers' private data wi
...
 
 
 <h5 align="center"> 
-    <img src="packages/syft/docs/img/big-picture.png" alt="big-picture-overview" width="400">
+    <img src="packages/syft/docs/img/big-picture.png" alt="big-picture-overview" width="720">
 </h5>
 
 
-
 # Deployment
```
Comment (abhiwalia15) [inline README_NEW.md:50]: @madhavajay, I needed your input on the Deployment section. Right now, the deployment section is not the priority. Hence, we were thinking of moving everything from the current README into separate files and giving links to them here. Later, we can make separate tutorials for each deployment method and have them more organized. Let me know what you think about this.

Also, the above quickstart section is being worked on by @shubham, and once that part is added, the text part above will give more context. Right now, it looks only text-text, which I believe was not our intention, but after discussing it with @irina, we think the part makes sense to explain the infographic. 

However, let us know, and we can make changes anytime. 

Thank you :)

**Comment by shubham3121** [comment]
This README looks great, just a different world !!! 😍

**Comment by ShubhamPalriwala** [comment]
Hey, I was wondering that changing the "Copying Data" to "Accessing data" in the pilot image might make more sense?
Screenshot_2022-08-01-14-55-17_1920x1080

**Comment by madhavajay** [comment]
> Hey, I was wondering that changing the "Copying Data" to "Accessing data" in the pilot image might make more sense? Screenshot_2022-08-01-14-55-17_1920x1080

You are accessing it though.... you're just not copying it. This is a surprisingly complicated thing to describe because we actually lack words in the English language to describe "utilizing" something without "seeing", "viewing", "accessing", "inspecting", "examining", "observing".

I think that `access` implies too much capability for the individual to both `enter` and `locate` data: https://en.wiktionary.org/wiki/access 

Having said that this is by no means final, so ill bing up your suggestion when we go for a copy review before merging.

**Comment by madhavajay** [comment]
Phenomenal work @abhiwalia15 🙌🏽

**Pull request merged**